ui/src/jobs/components/modal/DeleteJobModal.js

```diff
@@ -45,7 +45,8 @@ export const DeleteJobModal = ({
   return (
     <ConfirmationModal
       title={isActiveJobStatus(job.status) ? 'Delete Ensembling Jobs' : 'Terminate Ensembling Jobs'}
-      onConfirm={submitForm}
+      onCancel={() => setDeleteConfirmation("")}
+      onConfirm={(arg) => {submitForm(arg); setDeleteConfirmation("")}}
```

Comment:
There is no `modalClosed` function here, so we're just calling `setDeleteConfirmation` directly to reset the deletion text field.